packages/flutter/test/foundation/stack_frame_test.dart

```diff
@@ -47,6 +47,28 @@ void main() {
       webStackTraceFrames,
     );
   });
+
+  test('Parses ...',  () {
+    expect(
+      StackFrame.fromStackTraceLine('...'),
+      StackFrame.stackOverFlowElision,
+    );
+  });
+
+  test('Live handling of Stack Overflows', () {
+    void overflow(int seed) {
+      overflow(seed + 1);
+    }
+    bool overflowed = false;
+    try {
+      overflow(1);
+    } on StackOverflowError catch (e, stack) {
+      overflowed = true;
+      final List<StackFrame> frames = StackFrame.fromStackTrace(stack);
+      expect(frames.contains(StackFrame.stackOverFlowElision), true);
+    }
+    expect(overflowed, true);
+  }, skip: isBrowser);
```

Comment:
For future me: The browser test harness doesn't seem to be able to handle this, but the VM does.